pxp/request_response.md

```diff
@@ -200,20 +237,42 @@ The *transaction_id* should be used as a reference to the remote action.
 |------|------|------------
 | transaction_id | string | free format id of the request/response transaction
 
-### Error Message
+### RPC Error Message
```

Comment:
The title becomes more precise, as the name in the message type is "rpc_error_message".

As for the name itself, you must be able to distinguish between this and the PCP error message. So it cannot be simply "error message". "error response" could work (using "response" instead of "error", as you'll be always using this as a response). I don't like "rpc" too.

Anyway, this is probably not the right place to discuss about this.